htdocs/mri/jiv/get_file.php

```diff
@@ -131,6 +137,12 @@
     $MimeType         = 'image/vnd.nrrd';
     $DownloadFilename = basename($File);
     break;
+case 'DICOMTAR':
+    // ADD case for DICOMTAR
+    $FullPath         = $imagePath . '/' . $File;
```

Comment:
@christinerogers 

$imagePath can eventually be changed to get the tarchiveLibraryDir from Config table (once https://github.com/aces/Loris/pull/2649 is merged).